winrm/client.go

```diff
@@ -2,19 +2,25 @@ package winrm
 
 import (
 	"bytes"
-	"crypto/tls"
 	"crypto/x509"
 	"fmt"
 	"io"
-	"net/http"
+	"launchpad.net/gwacl/fork/http"
```

Comment:
The standard library does not support TLS renegotiation, which is required by winrm/azure. This fork maintained and used by the Juju project.